web/serve.py

```diff
@@ -20,12 +21,18 @@ class User(db.Model):
     name = db.Column(db.String(200))
 
 db.create_all()
+example_user = User(id=1, name="Philip Sterne")
+db.session.merge(example_user)
 db.session.commit()
 
-
 @app.route('/')
 def index():
     return render_template('index.html')
 
+@app.route('/users')
+def users():
+    users = User.query.all()
+    return render_template('users.html', users=users)
+
 if __name__ == '__main__':
-    app.run(host='0.0.0.0', port=5000)
+    app.run()
```

Comment:
I removed the `0.0.0.0` (which exposes the server to the rest of the network).
Port is now a configurable environment variable.